### DISCUSSION TITLE
An issue on iOS 10.3.3.

### DISCUSSION THREAD

**Discussion by tjdam** [react-hook-form/react-hook-form#1926] [Q&A]
Hi!
I switched to RHF from Formik recently and I'm just having a great time with it, it works so great.

But today I had the chance of testing it on an iPad 4th generation and found out that on every browser I tried in it, it gave me a Unhandled Runtime Error, ReferenceError: Can't find variable: VALIDATION_MODE.

I tried to replace back the Formik version and it works normally. I don't really want to go back to Formik though 😬

Does anyone have any idea of what could it be?

Thanks :)

**Comment by bluebill1049** [comment]
can you please take a screenshot? I don't think this is problem with RHF, probably an issue with your bundler. happy to see if there is a solution for this. are you sing create react app or your own webpack?

**Reply by tjdam** [reply]
I'm using Next.js, just updated to 9.4.5-canary.13, but I was on 9.3 earlier today.

For whatever reason, the error appears in the most annoying way possible: the call stack shows in this very small window and you can only really read one line at the time:
![IMG_0005](https://user-images.githubusercontent.com/57987742/85199447-0f52a580-b333-11ea-8212-36587f88a8cb.PNG)

But this is the entire log:
```
1 of 1 unhandled error

Unhandled Runtime Error
ReferenceError: Can't find variable: VALIDATION_MODE

Call Stack

dispatchEvent
[native code]

performSyncWorkOnRoot
[native code]

dispatchDiscreteEvent
[native code]
```

Thanks!

**Reply by bluebill1049** [reply]
which version of RHF?

**Reply by tjdam** [reply]
5.7.2

**Reply by bluebill1049** [reply]
i would recommend do a `npm clean` and remove all your node module as this is really strange.

**Reply by tjdam** [reply]
Just gave it a go, but no luck.
Thank you for your time, anyways!
And thank you for the great work with the library :)

**Reply by bluebill1049** [reply]
That's all good, sorry I couldn't help much with this one...
